packages/@okta/vuepress-site/docs/guides/oin-sso-overview/index.md

```diff
@@ -0,0 +1,94 @@
+
+### Example of a partner integration journey with Okta
+
+Erika is an application developer at Acme, a technology partner with Okta. Acme is looking to leverage the OIN as a way for their customers to adopt and incorporate Acme’s application to the customer’s existing Okta tenant. This will allow Acme’s customers to add Acme’s app to their existing identity infrastructure with minimal integration resources.
+
+1. Erika builds the Acme-Okta integration, doing the heavy lifting so that their customers don’t have to.
+2. Erika documents the required configuration steps for the customer admin.
+3. Erika submits the integration and corresponding documentation for the Okta OIN team to verify and review.
+4. After approval, Acme’s app is published on the OIN.
+
+With a pre-built Acme-Okta integration, Acme avoids additional support staff required for each individual customer integration.
+
+### Example of an identity admin journey with Okta
+
+Ali is an IT admin at Initech. Initech is looking to add Acme's application into their existing Okta identity infrastructure.
+
+1. Ali finds the Acme app in the OIN catalog. Since Acme is in the OIN, Ali knows that he can trust Acme to be securely incorporated into their existing Okta-managed SSO with minimal integration effort required.
+
+2. Ali adds the Acme app integration from the Okta Admin Console. Ali follows the instructions provided by Acme to configure the app integration.
+
+3. Ali configures the Okta authentication policy and the group of Initech employees who will have access to the Acme app.
+
+4. Ali tests signing in to the Acme app with existing Okta credentials to verify the authentication flow.
+
+Initech's group of employees with privileged access can sign in to the Acme app with their existing Okta credentials and no additional Acme app registration is required.
+
+### Example of a workforce user Single Sign-On journey with Okta
+
+Ramon is an Initech employee with privileges to the Acme app.
+
+1. Ramon starts his work day. In his web browser, he clicks the Okta browser extension and selects his email app which loads in a new tab.
+2. Ramon’s company has an Okta sign-on policy, which requires each employee to verify their identity every 12 hours. Since it’s been more than 12 hours since he was last working, he is prompted to enter his Okta username and password.
+3. Ramon’s company has also enabled Okta multifactor authentication. After successfully entering his credentials, a push notification is sent to the Okta Verify app on his phone. Ramon taps his phone to verify his identity. Ramon can now access his email.
+4. Next, Ramon goes to his Okta browser extension and selects the Acme app. Since Ramon started a session less than 12 hours ago, he is taken directly into the app without needing to sign-in again. In fact, Ramon can access all the Okta-integrated apps that he has privileges to without signing in again because he already has an authenticated session with Okta.
```

Comment:
```suggestion
4. Next, Ramon goes to his Okta browser extension and selects the Acme app. Since Ramon started a session less than 12 hours ago, he is taken directly into the app without needing to sign in again. In fact, Ramon can access all the Okta-integrated apps that he has privileges to without signing in again because he already has an authenticated session with Okta.
```